### PR TITLE
Pass the outcomes title to rubrics

### PR DESCRIPTION
**Pull request by MitchelPaulin**
Rubrics needs to know what the current outcome term is, passing that information along

### CODE REVIEW

File: components/right-panel/consistent-evaluation-rubric.js [hunk 1]
```diff
@@ -322,6 +326,7 @@ class ConsistentEvaluationRubric extends LocalizeConsistentEvaluation(RtlMixin(L
 						@d2l-rubric-finished-saving=${this._syncActiveRubricChangeHandler}
 						?ce-popout=${this.isPopout}
 						?show-published-warning=${this.evaluationPublished}
+						outcomes-title-text=${this.outcomeTerm}
```
Comment (vix633) [inline components/right-panel/consistent-evaluation-rubric.js:329]: Do we want to handle this if it has no value?

Comment (MitchelPaulin) [reply components/right-panel/consistent-evaluation-rubric.js:329]: Can it have no value? To be safe I could pass it through a localize call which has a default https://github.com/BrightspaceHypermediaComponents/consistent-evaluation/blob/master/lang/en.js#L102

Comment (MitchelPaulin) [reply components/right-panel/consistent-evaluation-rubric.js:329]: Actually that might break if we try to re-localize it on our end, I'll just default it to "Standards'

File: components/right-panel/consistent-evaluation-rubric.js [hunk 2]
```diff
@@ -177,18 +181,19 @@ class ConsistentEvaluationRubric extends LocalizeConsistentEvaluation(RtlMixin(L
...
+			if (rubricInfos && rubricInfos.length === 1) {
 				this._dispatchInitScoringRubric();
 			}
 			if (this.activeScoringRubric !== undefined
 				&& this.activeScoringRubric !== 'undefined'
 				&& (changedProperties.get('activeScoringRubric') !== undefined
 				|| changedProperties.get('activeScoringRubric') !== 'undefined')) {
 				if (this.activeScoringRubric === 'null' || this.activeScoringRubric === this.unassignedActiveScoringRubricId) {
-					this.activeScoringRubric = changedProperties.get('rubricInfos').filter(rubric => rubric.rubricScoringMethod !== 0)[0].rubricId;
+					this.activeScoringRubric = rubricInfos.filter(rubric => rubric.rubricScoringMethod !== 0)[0].rubricId;
```
Comment (MitchelPaulin) [inline components/right-panel/consistent-evaluation-rubric.js:196]: Kept getting console errors dumped here, its possible for the key `rubricInfos` to be in the map but its actual value to be undefined, adding another check for that here

**Review by JamieTran — APPROVED** [review]

**Pull request merged**